change/@ni-nimble-components-23b196b3-58c9-410d-8c7b-277b9547c270.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
1. The storybook docs for the old primary button class were updated to mention appearance-variant but otherwise unchanged.
2. The matrix tests already covered primary buttons (by having it as a separate dimension so all cases are shown, not just supported). I think keeping all the permutations is the safe bet so I'd lean to not changing it in this PR, but If we want to change the matrix test to make it easier to maintain, etc, we can do it in a follow-up PR. Not clear to me what the desired goal would be yet.